minigun/minigun/spmat.h

```diff
@@ -27,6 +27,25 @@ struct Csr {
 using IntCsr = Csr<int32_t>;
 using LongCsr = Csr<int64_t>;
 
+template <typename Idx>
+struct Coo {
+  IntArray1D<Idx> row;
+  IntArray1D<Idx> column;
+};
```

Comment:
Add `num_rows` and `num_cols` for both Csr and Coo?

Reply:
DONE.